docs/contracts/index.md

```diff
@@ -67,6 +72,20 @@ Finally, you can set the `DEBUG_CONTRACTS` constant to `True` to enable debuggin
 .. autodata:: python_ta.contracts.DEBUG_CONTRACTS
 ```
 
+## CLI
+
+The `python_ta.contracts` CLI can execute a file as `__main__` with contracts enabled. This option
+requires there to be an `if __name__ == "__main__":` so PythonTA knows when and where to
+start tracking contracts from.
+
+The syntax for this command is: `python -m python_ta.contracts [OPTIONS] FILE [EXTRA_MOD_NAMES]`
```

Comment:
You don't need to specify the options here. Instead, show just the basic usage, and then say that the user can run `python_ta.contracts --help` for information about the command-line options.

Reply:
Oh and say that the user might need to use `python3` instead of `python` depending on their system.